coalib/bears/Bear.py

```diff
@@ -274,6 +281,284 @@ def log_message(self, log_message, timestamp=None, **kwargs):
+                    elif line.strip(' \n\t') != '':
+                        table_header.append(line.strip())
+                channel_values = [x for x in [y.split(' ') for y in
+                                              req_lines] if x and not x == '\n']
+
+                table_data = []
+                final_table_data = []
+                for x in channel_values:
+                    table_data.append(list(filter(lambda a: a != '', x)))
+                for x in table_data:
+                    start = ''
+                    flag = 0
+                    row = []
+                    for y in x:
+                        if y.startswith('{'):
+                            flag = 1
+                        if flag == 1:
+                            start = start + y + ' '
+                        elif y != '':
+                            t = ' '.join(y.split())
+                            row.append(t.strip())
+                    if start != '':
+                        row.append(start.strip())
+                    final_table_data.append(row)
+                colored_table_data = []
+                colors = ['red', 'white', 'blue', 'yellow', 'magenta', 'green']
+                linter_table = []
+                linter_table.append(final_table_data[0])
+                if u'linter' in str(self):
+                    for i in final_table_data:
+                        if u'linter'.title() in i[5] or str(self.__class__.__name__) in i[5]:
+                            linter_table.append(i)
+
+                if u'linter' in str(self):
+                    final_table_data = linter_table
+                else:
+                    final_table_data = final_table_data if check_trim else (
+                        final_table_data[:15])
+                for x in final_table_data:
+                    row = []
+                    for index, y in enumerate(x):
+                        row.append(colored(y, colors[index]))
+                    colored_table_data.append(row)
+                table = AsciiTable(colored_table_data)
+                for line in table_header:
+                    print(line)
+                cprint(table.table)
+        finally:
+            remove(path)
+
+    def setup_profiler(self, *args, **kwargs):
+        profile_output =kwargs.get('profile_output', False)
+        profile_dump = kwargs.get('profile_dump', False)
+        profile_commands = kwargs.get('profile_commands', False)
+        if str(profile_output) is not u'False':
+            profile_bears = self.parenthesis_split(str(profile_output))
+        if str(profile_dump) is not u'False':
+            profile_bears = self.parenthesis_split(str(profile_dump))
+        if str(profile_commands) is not u'False':
+            profile_bears = self.parenthesis_split(str(profile_commands))
+        if (str(profile_output) is u'False') and (str(profile_dump) is u'False') and (str(profile_commands) is u'False'):
```

Comment:
Line is longer than allowed. (121 > 80)

*Origin: LineLengthBear, Section: `linelength`.*